python/sglang/srt/layers/attention/triton_ops/decode_attention.py

```diff
@@ -534,7 +545,7 @@ def _decode_grouped_softmax_reducev_fwd(
     BLOCK = 128
     batch, head_num = b_seq_len.shape[0], logits.shape[0]
     kv_group_num = logits.shape[0] // v_buffer.shape[1]
-    BLOCK_H = max(16, triton.next_power_of_2(kv_group_num))
+    BLOCK_H = max(16, min(64, triton.next_power_of_2(kv_group_num)))
```

Comment:
just curious - under what case will this OOM? and is it OOM on SRAM?

Reply:
For deepseek-v2，the head_num is 128 if we use DP. I added this situation in the unit test.